e2etest/bgp_test.go

```diff
@@ -95,6 +130,15 @@ var _ = ginkgo.Describe("BGP", func() {
 	ginkgo.Context("type=Loadbalancer", func() {
 		var allNodes *corev1.NodeList
 		ginkgo.BeforeEach(func() {
+			var peers []peer
+			for _, c := range frrContainers {
+				peers = append(peers, peer{
+					Addr:  c.Ipv4,
+					ASN:   c.RouterConfig.ASN,
+					MyASN: c.NeighborConfig.ASN,
```

Comment:
did u get the above swapped ?

Reply:
From the speaker's point of view he's the container's neighbor (MyASN should come from neighborconfig) and ASN,Addr corresponds to the RouterConfig and container's IP.

Reply:
I understand ur explaintion IMO this will be source of continous confusion we should just consider them as two separate routers and use same syntax its known remote-as of neghibor has the match router instance

Reply:
Could you explain why it's confusing? The containersConf describes which containers we want the tests to setup and with which configuration, for them the NeighborConfig is reflected in the bgp conf file (they need to know who are their neighbors), and the RouterConfig is how they should configure/present themselves (reflected in the daemons conf file).

Reply:
never mind its clear now Thanks